```
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20210_20210&logView=flowAware)  [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:testCompile (default-testCompile) on project fe-core: Compilation failure: Compilation failure:
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20211_20211&logView=flowAware)  [ERROR] /root/doris/fe/fe-core/src/test/java/org/apache/doris/nereids/parser/EncryptSQLTest.java:[470,13] cannot find symbol
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20212_20212&logView=flowAware)  [ERROR]   symbol:   variable Config
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20213_20213&logView=flowAware)  [ERROR]   location: class org.apache.doris.nereids.parser.EncryptSQLTest
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20214_20214&logView=flowAware)  [ERROR] /root/doris/fe/fe-core/src/test/java/org/apache/doris/nereids/parser/EncryptSQLTest.java:[502,13] cannot find symbol
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20215_20215&logView=flowAware)  [ERROR]   symbol:   variable Config
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20216_20216&logView=flowAware)  [ERROR]   location: class org.apache.doris.nereids.parser.EncryptSQLTest
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20217_20217&logView=flowAware)  [ERROR] -> [Help 1]
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20218_20218&logView=flowAware)  [ERROR]
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20219_20219&logView=flowAware)  [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20220_20220&logView=flowAware)  [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20221_20221&logView=flowAware)  [ERROR]
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20222_20222&logView=flowAware)  [ERROR] For more information about the errors and possible solutions, please read the following articles:
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20223_20223&logView=flowAware)  [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20224_20224&logView=flowAware)  [ERROR]
[03:10:57 ](http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/938433?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&showLog=938433_20225_20225&logView=flowAware)  [ERROR] After correcting the problems, you can resume the build with the command
```

FE Compile error

--- a/fe/fe-core/src/test/java/org/apache/doris/nereids/parser/EncryptSQLTest.java
+++ b/fe/fe-core/src/test/java/org/apache/doris/nereids/parser/EncryptSQLTest.java
@@ -467,7 +467,6 @@ public class EncryptSQLTest extends ParserTestBase {
                 })) {
             ctx.setEnv(env);
             ctx.setCurrentUserIdentity(UserIdentity.ROOT);
-            Config.enable_nereids_load = true;
             // testing for https://github.com/apache/doris/issues/62140
             String sql = "CREATE USER 'test_user62140'@'%' IDENTIFIED BY '123456'";
             String res = "CREATE USER 'test_user62140'@'%' IDENTIFIED BY '*XXX'";
@@ -499,7 +498,6 @@ public class EncryptSQLTest extends ParserTestBase {
                 })) {
             ctx.setEnv(env);
             ctx.setCurrentUserIdentity(UserIdentity.ROOT);
-            Config.enable_nereids_load = true;
             // testing for https://github.com/apache/doris/issues/62140
             String sql = "ALTER USER 'test_user62140'@'%' IDENTIFIED BY '123456'";
             String res = "ALTER USER 'test_user62140'@'%' IDENTIFIED BY '*XXX'";

